lib/prepare_security.js

```diff
@@ -48,15 +51,16 @@ export default class SecurityReleaseSteward {
     cli.ok('Done!');
   }
 
-  async createVulnerabilitiesJSON(req, release, { cli }) {
+  async createVulnerabilitiesJSON(req, release, releaseDate, { cli }) {
```

Comment:
I think this PR does more than prompt dependency updates, right? You are managing the releaseDate

Reply:
Yeah it adds the releaseDate in the vulnerabilities.json, but its a minor thing

Reply:
may you split it into a different commit at least?

Reply:
removed it, will do in a different pr